doc/storagedriver/azure.md

```diff
@@ -10,6 +10,7 @@ The following parameters must be used to authenticate and configure the storage
 * `accountname`: Name of the Azure Storage Account.
 * `accountkey`: Primary or Secondary Key for the Storage Account.
 * `container`: Name of the root storage container in which all registry data will be stored. Must comply the storage container name [requirements][create-container-api].
+* `domain`: (optional) Domain name for the Storage Service. Defaults to `core.windows.net`. For example, `core.chinacloudapi.cn` would be the correct value for Azure in China and `core.usgovcloudapi.net` would be the correct value for Azure Government.
```

Comment:
Should this be called `host` or `addr`? We don't really use the word "domain" in this context. For example, below, it gets set to `DefaultBaseUrl` if not set. Looking at `DefaultBaseUrl`, it actually looks like it should be a "host".